### ISSUE TITLE
customize-the-serverless-iam-policy.md - suggestions

### ISSUE THREAD

**Issue by Lior-G** [AnomalyInnovations/serverless-stack-com#439]
under "An advanced IAM Policy template", recommending the following updates

* update (twice)

Why: **PATCH** is needed for any deployment after the first one

```
"apigateway:GET",
"apigateway:POST",
"apigateway:PUT",
"apigateway:DELETE"
```
to
```
"apigateway:GET",
"apigateway:PATCH",
"apigateway:POST",
"apigateway:PUT",
"apigateway:DELETE"
```

* update

WHY: ensure code only has access to S3 permissions within project

```
"arn:aws:s3:::*"
```
to
```
"arn:aws:s3:::<service_name>*"
```

* update

WHY: ensure code only has access to S3 permissions within project

```
"arn:aws:s3:::*/*"
```
to
```
"arn:aws:s3:::<service_name>*/*"
```

**Comment by jayair** [comment]
Yeah these make sense. Do you mind editing the chapter and submitting a PR?

**Comment by Lior-G** [comment]
> Yeah these make sense. Do you mind editing the chapter and submitting a PR?

Jay:
sure, no problem.
Looking at your CONTRIBUTING.md, don't see if there is a branching naming convention, or steps to properly submit a PR.
Is there any? if so, can you please point me to it?

**Comment by jayair** [comment]
For this case you can simply edit this chapter through the GitHub web interface and submit a PR. Does that make sense?

**Comment by Lior-G** [comment]
already done, and PR has already been merged

cheers

Lior
